### ISSUE TITLE
Webpacker overlaps SASS compilation

### ISSUE THREAD

**Issue by kstratis** [rails/webpacker#1337]
Using `Rails 5.1`, whenever I was making a change in any of my javascript files under `app/javascript` and then hit the browser refresh button, the console would spit out a small `[Webpacker] Compiling…` line and the page would then render.

However any changes made to `.scss` files under `app/assets/stylesheets` were handled by sprockets alone even though `webpacker.yml` had the following line (no webpacker compiling step here):

```yml
# Additional paths webpack should lookup modules
resolved_paths: ['app/assets']
```

 ---

I recently upgraded to `Rails 5.2 rc1` and `Webpacker 3.3.0`. 

The "problem" is that now any changes made in files under `app/assets/stylesheets` are also picked up by `Webpacker` (`[Webpacker] Compiling…`) which I think makes the entire compilation cycle slower.

In short: 
- Is webpack changed in any way so that it picks up sass by default?
- If so, is there a way to disable this and revert its behavior to the way it was in Rails 5.1?
- Why did that happen?

**Comment by maschwenk** [comment]
@gauravtiwari this was most likely caused by my PR #816

**Comment by gauravtiwari** [comment]
@maschwenk Yeah looks like it but that's fine behaviour. Although I guess would make sense to let user define globs since now people can't do `app/assets/**/*.js`

@kstratis What happens if you change resolved paths to: `resolved_paths: ['app/assets/javascripts']` or specific asset folder?

**Comment by gauravtiwari** [comment]
For performance, you don't want to include everything in the modules namespace so, only add the paths you need to be resolved.

**Comment by kstratis** [comment]
@gauravtiwari: Indeed setting `resolved_paths: ['app/assets/javascripts']` alleviates the "problem".
Globs would be nice to have @maschwenk !

Thank you for your prompt responses.
Please feel free to close the issue since this is expected behavior.
